src/components/CopyIconWithAddress/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { useRef } from 'react';
+import { useMediaQuery } from 'usehooks-ts';
```

Comment:
Didnt we already have this ?

Reply:
what exactly you mean

Reply:
CopyIconWithAddress

Reply:
where?

Reply:
![2024-03-13_14-12](https://github.com/liberland/liberland_frontend/assets/29551049/65becf74-5f92-4a3a-b9b0-a031ceffef35)